raiden-dapp/src/plugins/service-worker-assistant-plugin.ts

```diff
@@ -0,0 +1,20 @@
+import path from 'path';
+import Vue from 'vue';
+
+const SERVICE_WORKER_SCRIPT = path.join(process.env.BASE_URL ?? '/', 'service-worker.js');
+
+/**
+ * @param _Vue - global Vue instance to act on
+ * @param _options - eventual configuration for the plugin (ignored)
+ */
+export async function ServiceWorkerAssistantPlugin(
+  _Vue: typeof Vue,
+  _options?: null,
+): Promise<void> {
+  const serviceWorkerAreSupported = 'serviceWorker' in navigator;
```

Comment:
This should either be `serviceWorkerIsSupported` or `serviceWorkersAreSupported`

Reply:
:heavy_check_mark: